Resources/Prototypes/Entities/Objects/Specific/Medical/hypospray.yml

```diff
@@ -155,7 +152,7 @@
         - ReagentId: TranexamicAcid
           Quantity: 3
   - type: Tag
-    tags: []
```

Comment:
these prototypes have this to remove trash tag, which isnt needed anymore
put Medipen tag on a filled medipen abstract prototype they all inherit